File: FastScript/Demos/Main/Samples/JScript/is.js
//**************************}
//{ FastScript v1.0          }
//{ 'is' operator demo       }
//{**************************/


  var o = new TObject;
  var p = new TPersistent;
  if (o is TObject)
    ShowMessage("o is TObject");
  if (p is TObject)
    ShowMessage("p is TObject");
  if (!(o is TPersistent))
    ShowMessage("o is not TPersistent");

File: FastScript/Demos/Main/Samples/BasicScript/is.vb
' FastScript v1.0          
' 'is' operator demo       


dim o, p

  o = new TObject
  p = new TPersistent
  if o is TObject then
    ShowMessage("o is TObject")
  end if
  if p is TObject then
    ShowMessage("p is TObject")
  end if
  if not (o is TPersistent) then
    ShowMessage("o is not TPersistent")
  end if
